modules/installation-azure-service-principal.adoc

```diff
@@ -177,14 +176,16 @@ output. You need these values during {product-title} installation.
 +
 [source,terminal]
 ----
-$ az ad sp create-for-rbac --role Contributor --name <service_principal> <1>
+$ az ad sp create-for-rbac --role Contributor --name <service_principal> \ <1>
+  --scopes /subscriptions/<id>
```

Comment:
--scopes /subscriptions/<subscriptionId>
<2> Replace `<subscriptionId>` with the id of the subscription